### PR TITLE
Deprecate DriveDistanceAuto and configure AutoBuilder

### PR DESCRIPTION
**Pull request by renatodellosso**

### CODE REVIEW

File: RobotCode/src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -12,20 +14,26 @@ public class RobotContainer
...
 		addAutonomousOptions();
 
 		// Configure the button bindings
 		configurePrimaryBindings();
 		configureSecondaryBindings();
 	}
 
+	private void registerNamedCommands()
+	{
+
```
Comment (seankid) [inline RobotCode/src/main/java/frc/robot/RobotContainer.java:34]: if there is supposed to be something here change it. otherwise is fine

**Review by seankid — APPROVED** [review]
looks good overall

File: RobotCode/src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (renatodellosso) [reply RobotCode/src/main/java/frc/robot/RobotContainer.java:34]: That method is intentionally empty. It'll be expanded later as we add commands.

**Review by Tr01ler — APPROVED** [review]
Looks good to me.

**Pull request merged**